core/fragment.c

```diff
@@ -4098,7 +4098,7 @@ fragment_add_ibl_target_helper(dcontext_t *dcontext, fragment_t *f,
 }
```

Comment:
nit in the PR title: we generally don't have `fix:` prefixes in the title.

s/fix: return/Fix return/